backend/app/usecase/repository/shortlink_fake.go

```diff
@@ -105,6 +105,22 @@ func (s ShortLinkFake) UpdateShortLink(oldAlias string, shortLinkInput entity.Sh
 	}, nil
 }
 
+// DeleteShortLink deletes an existing user short link.
+func (s ShortLinkFake) DeleteShortLink(input entity.ShortLinkInput) error {
+	alias := input.GetCustomAlias("")
+
+	if alias == "" {
+		return errors.New("empty alias")
+	}
```

Comment:
```suggestion
```
Why we need this? If the alias is empty then when looking at the map it cannot be found.

Reply:
When it's used for testing it should return an error just like the non mock version